parquet-hadoop/README.md

```diff
@@ -211,6 +211,23 @@ conf.set("parquet.bloom.filter.enabled#column.path", false);
 
 ---
 
+**Property:** `parquet.bloom.filter.adaptive.enabled`  
+**Description:** Whether to enable writing adaptive bloom filter.  
+If it is true, the bloom filter will be generated with the optimal bit size 
+according to the number of real data distinct values. If it is false, it will not take effect.
+Note that the maximum bytes of the bloom filter will not exceed `parquet.bloom.filter.max.bytes` configuration.
```

Comment:
nit: would be good to explain the generated bloom filter will not be efficient if `parquet.bloom.filter.max.bytes` is too small.

Reply:
done